view/migrations/20220908223928_notes.sql

```diff
@@ -34,11 +34,17 @@ CREATE TABLE spendable_notes (
     height_spent            BIGINT 
 );
 
+CREATE INDEX spendable_notes_by_nullifier_idx ON spendable_notes (
+    nullifier
+);
+
+CREATE INDEX spendable_notes_by_source_idx ON spendable_notes (
+    source
+);
```

Comment:
This is otherwise unrelated, but it seemed better to have separate indexes for nullifier / source, both of which are random data, than the more meaningful / structured data in the other index.